### PR TITLE
Feature/docs 2168 api policy enforcement update

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/user-guides/search-and-filters/use-search.md [hunk 1]
```diff
@@ -398,7 +398,7 @@ You can find the node UUID in the **Nodes** section, [node details](../../user-g
 
 ### Search by specification
 
-To get the list of events related to specific [specification policy violations](../../api-policy-enforcement/overview.md), in the search field specify `spec:'<SPECIFICATION-NAME>'`.
+To get the list of events related to specific [specification policy violations](../../api-policy-enforcement/overview.md), in the search field specify `spec:'<SPECIFICATION-ID>'`.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/search-and-filters/use-search.md:401]: how to get a spec ID?

Comment (AnastasiaTWW) [reply docs/latest/user-guides/search-and-filters/use-search.md:401]: imagine you are a user

File: docs/latest/api-policy-enforcement/setup.md [hunk 2]
```diff
@@ -13,7 +13,15 @@ This article describes how to enable and configure your API protection based on
 
     ![Upload specification](../images/api-policies-enforcement/specificaton-upload.png)
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/setup.md:15]: there are warnings in 4.10 build

File: docs/latest/api-policy-enforcement/setup.md [hunk 3]
```diff
@@ -13,7 +13,15 @@ This article describes how to enable and configure your API protection based on
...
 
-## Step 2: Set actions for violations of policies
+## Step 2: Upload specification
+
+1. Click **Upload**. This starts upload.
+1. If specification is valid (OpenAPI 3.0 JSON or YAML with correct formatting), as uploading is finished, proceed to the next step.
+1. If some errors found, fix them and try to re-upload.
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/setup.md:20]: extra, UI well explains that

File: docs/latest/api-policy-enforcement/setup.md [hunk 4]
```diff
@@ -13,7 +13,15 @@ This article describes how to enable and configure your API protection based on
 
     ![Upload specification](../images/api-policies-enforcement/specificaton-upload.png)
 
-## Step 2: Set actions for violations of policies
+## Step 2: Upload specification
+
+1. Click **Upload**. This starts upload.
+1. If specification is valid (OpenAPI 3.0 JSON or YAML with correct formatting), as uploading is finished, proceed to the next step.
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/setup.md:19]: extra, the UI weel explains that

File: docs/latest/api-policy-enforcement/setup.md [hunk 5]
```diff
@@ -13,7 +13,15 @@ This article describes how to enable and configure your API protection based on
 
     ![Upload specification](../images/api-policies-enforcement/specificaton-upload.png)
 
-## Step 2: Set actions for violations of policies
+## Step 2: Upload specification
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/setup.md:16]: just keep the previous step as "Upload specification". 

in the end, add "Note that you will not be able to start configuring API policy enforcement based on the specification, until its file is successfully uploaded."

File: docs/latest/api-policy-enforcement/setup.md [hunk 6]
```diff
@@ -22,12 +30,13 @@ This article describes how to enable and configure your API protection based on
         * Before using the specification for policy enforcement, it is recommended to use it for searching the rogue (shadow, zombie and orphan) APIs using API Discovery. This way you will be able to understand how much your specification differs from the actual requests of your clients.
 
 1. Select **Use for API specification-based policy enforcement**. Options are displayed.
-1. Specify **URI** to activate policy violation actions only for requests sent to certain endpoints.
+1. Specify host or endpoint for which you want to activate policy violation actions.
 
+    * This field is required.
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/setup.md:35]: it is obvious from the UI

File: docs/latest/api-policy-enforcement/viewing-events.md [hunk 7]
```diff
@@ -6,11 +6,11 @@ As soon as you [uploaded](setup.md) your API specification to be used for specif
 
 To monitor trends in policy violations, check the number of specification violations in Wallarm Console under **API Specifications** → your specification → **Policy violations** column. This data provides insights for the past 7 days.
 
-You can click this number to see details in the **Events** section.
+You can click this number to see details in the **Attacks** section.
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/viewing-events.md:9]: there are more **Events** entries in the included files

File: docs/latest/api-policy-enforcement/viewing-events.md [hunk 8]
```diff
@@ -6,11 +6,11 @@ As soon as you [uploaded](setup.md) your API specification to be used for specif
...
 
 ## Analysis on requests that violate policies 
 
-In the **Events** section, use the **Specification** filter or the `spec:'<SPECIFICATION-NAME>'` [search tag](../user-guides/search-and-filters/use-search.md#search-by-specification) to find all events related to the selected specification(s) policy violations.
+In the **Attacks** section, use the **Compare to...** filter or the `spec:'<SPECIFICATION-ID>'` [search tag](../user-guides/search-and-filters/use-search.md#search-by-specification) to find all events related to the selected specification(s) policy violations.
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/viewing-events.md:13]: they cannot use the spec tag as far as I could understand

File: docs/latest/user-guides/search-and-filters/use-search.md [hunk 9]
```diff
@@ -398,7 +398,7 @@ You can find the node UUID in the **Nodes** section, [node details](../../user-g
...
 
 ![Specification - use for API policy enforcement](../../images/api-policies-enforcement/api-policies-enforcement-events.png)
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/search-and-filters/use-search.md:404]: Grammarly on screenshots

File: docs/latest/api-policy-enforcement/setup.md [hunk 10]
```diff
@@ -13,7 +13,15 @@ This article describes how to enable and configure your API protection based on
...
-## Step 2: Set actions for violations of policies
+## Step 2: Upload specification
+
+1. Click **Upload**. This starts upload.
+1. If specification is valid (OpenAPI 3.0 JSON or YAML with correct formatting), as uploading is finished, proceed to the next step.
+1. If some errors found, fix them and try to re-upload.
+
+Note that you will not be able to start configuring API policy enforcement based on the specification, until its file is successfully uploaded.
+
+## Step 3: Set actions for violations of policies
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/setup.md:25]: the current documentatuon on API policu enforcement does not describe issues addressed by the feature but it should. business issues solved by this module

File: docs/latest/api-policy-enforcement/setup.md [hunk 11]
```diff
@@ -13,7 +13,15 @@ This article describes how to enable and configure your API protection based on
 
     ![Upload specification](../images/api-policies-enforcement/specificaton-upload.png)
 
-## Step 2: Set actions for violations of policies
+## Step 2: Upload specification
+
```
Comment (AnastasiaTWW) [inline docs/latest/api-policy-enforcement/setup.md:17]: update the texts and images once the texts in the UI changed

File: docs/latest/api-policy-enforcement/viewing-events.md (hunk 8 above)
Comment (MaximBashurov) [reply docs/latest/api-policy-enforcement/viewing-events.md:13]: They can once they know the ID.

**Pull request merged**